packages/gatsby/src/utils/webpack.config.js

```diff
@@ -527,16 +527,12 @@ module.exports = async (
           },
           priority: 30,
           minChunks: 1,
-          reuseExistingChunk: true,
         },
         commons: {
-          // only bundle non-async modules
-          chunks: `initial`,
```

Comment:
This wasn't doing what I was expecting, because of our components being async, we need this to be async as well or we won't bundle them into commons